### DISCUSSION TITLE
body dir attribute is always empty

### DISCUSSION THREAD

**Discussion by jpmckinney** [pradyunsg/furo#145] [Potential Issue] [answered]
Jinja2 seems to happily accept the undefined `{{ direction }}`. I assume the intention is to add this to `theme.conf` ?

I guess Sphinx renders Jinja2 without `StrictUndefined`, so there are no errors for undefined variables.

In any case, the template probably wants to wrap with `{% if direction is not none %} dir="{{ direction }}"{% endif %}`.

**Comment by pradyunsg** [comment, marked as answer]
https://github.com/pradyunsg/furo/commit/5aff99bea5b62afcf6b5d0af078a94c3a41e34ff fixed this.
